relnotes/resume.migration.md

```diff
@@ -0,0 +1,8 @@
+# Disallow direct resume in termination hooks
+
+It is now necessary to use `IScheduler.resume` method if one task has to be
```

Comment:
This commit should also edit the method comments added previously that said "you shouldn't do this". (It's now enforced, not merely prohibited by convention.)